assets/js/stripe-payment-request.js

```diff
@@ -424,6 +424,21 @@ jQuery( function( $ ) {
 			} );
 		},
 
+		debounce: function ( wait, func, immediate ) {
```

Comment:
This is probably not needed, but could you please lint/format this snippet?

Reply:
@RadoslavGeorgiev I tried linting and got no issues.Also, this feels like it should be part of the build script?

Reply:
Sorry, I had an issue with linter tried again and it's giving me lots of error across the whole file. Most of them really meaningless and not really sure this plugin is set up correctly for formatting and listing... I don't mind doing it but it's going to take a while to fix this all up. Going to defer it to another issue:
https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1209